features/controller_specs/headers.feature

```diff
@@ -0,0 +1,50 @@
+Feature: Headers
+
+  We recommend you to switch to request specs instead of controller specs if you want to set
+  headers in your call.
+  If you still want to set headers in controller specs, you can use `request.headers` as
+  mentioned bellow.
```

Comment:
Maybe also keep it as once sentence per line?

Reply:
I thought about it but the lines were pretty long in code. I push to staging then cut the line in the core where Relish was wrapping lines. :)